src/OWML.Launcher/App.cs

```diff
@@ -166,22 +166,32 @@ private void StartGame()
 
 			try
 			{
-				if (_owmlConfig.GamePath.Contains("Epic"))
+				void StartGameViaExe()
+				{
+					_writer.WriteLine("Starting game via exe...");
+					_processHelper.Start(_owmlConfig.ExePath, _argumentHelper.Arguments);
```

Comment:
maybe just move these lines to line 177 instead of calling this inline method?

Reply:
but this code is also called on line 193